### ISSUE TITLE
Zero loss_bbox ?

### ISSUE THREAD

**Issue by HyunJun-Jo** [rbgirshick/py-faster-rcnn#472]
I'm training the faster rcnn on my own dataset. But after 90000 iterations, I see that the training was abnormal, because when I run the demo file, the result was very strange. I think that it is from the very small loss_bbox. When I start to training, I get the zero loss_bbox which you can see below.

```
I0120 19:43:59.117846  7100 solver.cpp:229] Iteration 0, loss = 1.51499
I0120 19:43:59.117871  7100 solver.cpp:245]     Train net output #0: loss_bbox = 0 (* 1 = 0 loss)
I0120 19:43:59.117875  7100 solver.cpp:245]     Train net output #1: loss_cls = 0.214809 (* 1 = 0.214809 loss)
I0120 19:43:59.117894  7100 solver.cpp:245]     Train net output #2: rpn_cls_loss = 0.566602 (* 1 = 0.566602 loss)
I0120 19:43:59.117898  7100 solver.cpp:245]     Train net output #3: rpn_loss_bbox = 0.161475 (* 1 = 0.161475 loss)
I0120 19:43:59.117902  7100 sgd_solver.cpp:106] Iteration 0, lr = 0.0001
I0120 19:44:05.942761  7100 solver.cpp:229] Iteration 20, loss = 0.50546
I0120 19:44:05.942787  7100 solver.cpp:245]     Train net output #0: loss_bbox = 0 (* 1 = 0 loss)
I0120 19:44:05.942807  7100 solver.cpp:245]     Train net output #1: loss_cls = 0.000158322 (* 1 = 0.000158322 loss)
I0120 19:44:05.942812  7100 solver.cpp:245]     Train net output #2: rpn_cls_loss = 0.464551 (* 1 = 0.464551 loss)
I0120 19:44:05.942816  7100 solver.cpp:245]     Train net output #3: rpn_loss_bbox = 0.0581968 (* 1 = 0.0581968 loss)
I0120 19:44:05.942821  7100 sgd_solver.cpp:106] Iteration 20, lr = 0.0001
I0120 19:44:12.894174  7100 solver.cpp:229] Iteration 40, loss = 0.40116
I0120 19:44:12.894203  7100 solver.cpp:245]     Train net output #0: loss_bbox = 0 (* 1 = 0 loss)
I0120 19:44:12.894210  7100 solver.cpp:245]     Train net output #1: loss_cls = 0 (* 1 = 0 loss)
I0120 19:44:12.894229  7100 solver.cpp:245]     Train net output #2: rpn_cls_loss = 0.162291 (* 1 = 0.162291 loss)
I0120 19:44:12.894235  7100 solver.cpp:245]     Train net output #3: rpn_loss_bbox = 0.0115385 (* 1 = 0.0115385 loss)
I0120 19:44:12.894240  7100 sgd_solver.cpp:106] Iteration 40, lr = 0.0001
```

How can I solve this problem? I saw some solutions using padding the images before training, but to pad images, I need to change all the dataset also. It is a very time-consuming work. Is there any way to solve this problem? Thanks for your help in advance.

**Comment by HyunJun-Jo** [comment]
This is also come up when learning VOC2007. What is wrong with me?

**Comment by Timonzimm** [comment]
How did you solve the issue @HyunJun-Jo ?
